Tools/CMake/AMReXOptions.cmake

```diff
@@ -157,6 +157,20 @@ cmake_dependent_option( AMReX_DPCPP_ONEDPL "Enable DPCPP's oneDPL algorithms"  O
    "AMReX_GPU_BACKEND STREQUAL SYCL" OFF)
 print_option(  AMReX_DPCPP_ONEDPL )
 
+if (AMReX_DPCPP)
+   set(AMReX_INTEL_ARCH_DEFAULT "IGNORE")
```

Comment:
Do you think we could use a default here?

Reply:
We have something in GNU Make that uses `cat /sys/devices/cpu/caps/pmu_name` to determine the intel device type.  But that is not a reliable way for the new hardware.  There is also concern about having the new hardware name in the public repo.  We can revisit this later.  Let's leave this as is for now.

Reply:
Agreed.
When we approach this later on, we could try to keep the heuristic in a similar order as for CUDA.